server/routes/addPrisoner/prisonerDetailsController.test.ts

```diff
@@ -151,8 +151,8 @@ describe('Prisoner details', () => {
     const prisonerDetails = {
       firstName: 'John',
       lastName: 'Smith',
-      day: '02',
-      month: '04',
+      day: '2',
+      month: '4',
```

Comment:
Worked fine in the tests - because my test data assumed dates entered with leading zeros 🤦‍♂️!